addons/web/static/src/libs/bootstrap/bootstrap.scss

```diff
@@ -0,0 +1,10 @@
+.form-control {
+    // Overridden file input buttons theming
+    &::file-selector-button {
+        @include gradient-bg(darken($input-bg, 2%));
+    }
+
+    &:hover:not(:disabled):not([readonly])::file-selector-button {
+        background-color: darken($input-bg, 6%);
+    }
```

Comment:
Is this CSS based on something already existing in Bootstrap? I wonder the call the gradient-bg for the first rule but not for the second one?

Reply:
@qsm-odoo We have improved solution but we didn't get this comment, would you please elaborate?

Reply:
@qsm-odoo We declared the variables $form-file-button-bg and $form-file-button-hover-bg in the bootstrap_overridden.scss and bootstrap_overridden_frontend.scss file. Since this variable files are loaded separately in backend and frontend assets, for that reason we made the changes in bootstrap_overidden and bootstrap_overidden_frontend.

Reply:
Hello 👋

`bootstrap_overridden.scss` is loaded and shared by **both** bundles, while `bootstrap_overridden_frontend.scss` and `bootstrap_overridden_backend.scss` are used **only** for the frontend and backend, respectively.  

**If** we wanted to achieve the same visual result in both bundles, modifying `bootstrap_overridden.scss` would have been enough... however we don’t want that because, in the backend, the button is normally managed by the `color-scheme: [light/dark]` rule 😉

 @paru-odoo ,  I suspect that the reason why you had to touch both files, it's because you forgot to add the [`!default` flags](https://sass-lang.com/documentation/variables/#default-values), thus the compilation was "failing" somewhere 😉
The flag  is also needed to allow pro-users to customise, so it should be there.

Thus, in conclusion: 
- define **frontend specific** overrides in `bootstrap_overridden_frontend.scss` only
- use the `!default` flag